course/src/main/java/org/openedx/course/presentation/dates/CourseDatesScreen.kt

```diff
@@ -305,16 +212,6 @@ internal fun CourseDatesScreen(
             )
         }
 
-        val snackState = remember { SnackbarHostState() }
-        if (uiMessage is DatesShiftedSnackBar) {
-            val datesShiftedMessage = stringResource(id = R.string.course_dates_shifted_message)
-            LaunchedEffect(uiMessage) {
-                snackState.showSnackbar(
-                    message = datesShiftedMessage,
-                    duration = SnackbarDuration.Long
-                )
-            }
-        }
```

Comment:
The Shift Due Dates Snackbars are a bit different in the home tab and course tab.

When you're on the Dates tab, you don't need the "View All Dates" button because this button actually redirects the user to the Dates tab.